content/ai_exchange/content/docs/ai_security_overview.md

```diff
@@ -187,27 +187,34 @@ The groups of controls form a summary of how to address AI security (controls ar
       >([MONITORUSE](/goto/monitoruse/), [MODELACCESSCONTROL](/goto/modelaccesscontrol/), [RATELIMIT](/goto/ratelimit/))
     - 2c Adopt **new** IT security controls:  
       >([CONFCOMPUTE](/goto/confcompute/), [MODELOBFUSCATION](/goto/modelobfuscation/), [PROMPTINPUTVALIDATION](/goto/promptinputvalidation/), [INPUTSEGREGATION](/goto/inputsegregation/))
-3. Data scientists apply **data science security controls** risk-based :
```

Comment:
Please revert. The actors are important and the risk-based is about the applying, not about the controls